test/e2e/main screen/main-screen-start-competition-screen-cancel.yaml

```diff
@@ -0,0 +1,8 @@
+## GIVEN the application is on the main screen and competition not started
+## WHEN user clicks on competition start
+## THEN the competition popup start screen visible
+## WHEN user selects cancel
+## THEN popup screen closes
```

Comment:
one `THEN` would be preferred, because the `THEN` indicates the point where we won't perform more actions just validation(s). If a scenario contains more than one (non-consecutive) `THEN`, then it should be split into more scenarios